### ISSUE TITLE
Cannot override Review component classes

### ISSUE THREAD

**Issue by radavis** [CMSgov/design-system#817]
**Describe the bug**

```
<Review className="ds-u-border-bottom--0 ds-u-padding-y--0" />
```

generates the following HTML

```
<div class="ds-c-review ds-u-border-bottom--2 ds-u-padding-y--2 ds-u-justify-content--between ds-u-display--flex ds-u-border-bottom--0 ds-u-padding-y--0">...</div>
```

which achieves the desired result of eliminating the bottom border, but **does not** eliminate the y-axis padding.

**To Reproduce**
1. See above

**Expected behavior**
I would expect to be able to override the utility classes on this component to achieve the desired styling. 

**Screenshots**

![Screen Shot 2020-09-16 at 10 52 27 AM (3)](https://user-images.githubusercontent.com/1299034/93354271-cfed0580-f80a-11ea-82c6-d3d7c340d6b2.png)

**Desktop (please complete the following information):**
 - OS: macOS 10.15.6
 - Browser: Chrome
 - Version: 85

**Comment by bernardwang** [comment]
@radavis Thanks for submitting this! It does look like the `<Review>` component is incorrectly applying utility classes instead of including it in the component styles. This issue is tracked [here](https://jira.cms.gov/browse/WNMGDS-632)

**Comment by bernardwang** [comment]
This issue has been fixed in the latest v2.2.0 release. The review component doesnt generate utility classes by default anymore (the styles are now included inside `ds-c-review`), so your utility classes should work.
